exec/java-exec/src/main/java/org/apache/drill/exec/planner/common/DrillRelOptUtil.java

```diff
@@ -224,4 +226,64 @@ public Void visitInputRef(RexInputRef inputRef) {
     }
   }
 
+  public static boolean isLimit0(RexNode fetch) {
+    if (fetch != null && fetch.isA(SqlKind.LITERAL)) {
+      RexLiteral l = (RexLiteral) fetch;
+      switch (l.getTypeName()) {
+        case BIGINT:
+        case INTEGER:
+        case DECIMAL:
+          if (((long) l.getValue2()) == 0) {
+            return true;
+          }
+      }
+    }
+    return false;
+  }
+
+  public static boolean isProjectOutputRowcountUnknown(RelNode project) {
```

Comment:
Could you add javadoc for this utility function.

Reply:
Done